src/Octoshift/GithubApi.cs

```diff
@@ -4,6 +4,7 @@
 using System.Net;
 using System.Net.Http;
 using System.Threading.Tasks;
+using Amazon.S3.Model;
```

Comment:
We don't need this using here, it's probably a mistake.

Reply:
Addressed in https://github.com/github/gh-gei/pull/787/commits/f0def9ecd76d60ade5b2c4eddfae9913e85493b2! :tada: